src/ExtCmd.actor.cpp

```diff
@@ -1423,6 +1390,7 @@ struct GetDistinctCmd {
 	static Future<Reference<ExtMsgReply>> call(Reference<ExtConnection> ec,
 	                                           Reference<ExtMsgQuery> query,
 	                                           Reference<ExtMsgReply> reply) {
+		query->ns.second = query->query.getField("distinct").String();
```

Comment:
I believe you tapped the wire and made sure these key names are indeed the keys retrieved from `getFristKey`  😂

Reply:
Yep, checked that.